### PR TITLE
Support 'embeds' parameter to discord API

### PR DESCRIPTION
**Pull request by dogboydog**
Hi,

Support [embeds](https://birdie0.github.io/discord-webhooks-guide/structure/embeds.html) for discord notifications. Probably best for people to try to generate them in a previous step and set them there with ::set-env or ::set-output

Related #41 
Let me know any changes you would like.

### CODE REVIEW

**Comment by Ilshidur** [comment]
Thank you for your PR ! I currently don't have a lot of time to walk through your code. I think I'll be able to take a look at it next week.

**Comment by dogboydog** [comment]
Sounds good

File: entrypoint.js [hunk 1]
```diff
@@ -31,18 +31,28 @@ _.templateSettings.interpolate = /{{([\s\S]+?)}}/g;
...
+        embedsObject = JSON.parse(process.env.DISCORD_EMBEDS);
+     } catch (parseErr) {
+       console.error('Error parsing DISCORD_EMBEDS:' + parseErr);
+       process.exit(1);
+     }
+  }
   url = process.env.DISCORD_WEBHOOK;
   payload = JSON.stringify({
     content: message,
+    ...process.env.DISCORD_EMBEDS && { embeds: embedsObject},
```
Comment (Ilshidur) [inline entrypoint.js:55]:
```suggestion
    ...process.env.DISCORD_EMBEDS && { embeds: embedsObject },
```

**Review by Ilshidur — CHANGES_REQUESTED** [review]
Ok, I finally had time to inspect your PR.

I'm pretty happy with the result. I added some docs and fixed some "linting" issues. Thing is, I did not install a linter yet in this repo... :smile: 

If you're OK with my changes request, I'll merge it and immediately cut a release featuring your changes.

**Comment by dogboydog** [comment]
> 
> 
> Ok, I finally had time to inspect your PR.
> 
> I really like it. Also, I added some docs and fixed some "linting" issues. Thing is, I did not install a linter yet in this repo... 😄
> 
> If you're OK with my changes request, I'll merge it and immediately cut a release featuring your changes.

Sounds good to me! I committed your changes. Thanks for taking a look. After the merge, I might delete my fork. Once I ended up doing more customization logic for my discord notifications, it made sense to implement my own custom client in Python.  But I did test these changes which allowed `embeds` to be sent, so hopefully someone out there will get some use out of it.

**Review by Ilshidur — APPROVED** [review]
LGMT. Thanks again for this PR. I hope people will like it.

Let's go for the release.

**Pull request merged**